packages/cubejs-schema-compiler/adapter/MssqlQuery.js

```diff
@@ -133,6 +133,24 @@ class MssqlQuery extends BaseQuery {
       'date_to'
     )} date_to FROM (VALUES ${values}) ${this.asSyntaxTable} dates (date_from, date_to)`;
   }
+
+  newSubQuery(options) {
+    options = {...options, isSubQuery: true};
```

Comment:
Unfortunately, the name is deceiving here. Not only subqueries go through this routine. And also this is really base change and can't go to the MS SQL query.

Reply:
@paveltiunov Totally get it; I kept going back and forth between changing base and adding it to MSSQL when I was working on this; hated the way changing MSSQL 'felt' (repeating the code) and not knowing the code base very well, was reluctant to change Base.
I'll make the change to base and go from there.  What other use case is going through this routine, is it just the PreAggregation stuff?

Reply:
@JoshMentzer Yep. AFAIK it's only pre-aggregations.